gui/wxpython/lmgr/frame.py

```diff
@@ -263,6 +264,14 @@ def show_menu_errors(messages):
             self.GetMapDisplay().Raise()
         wx.CallAfter(self.Raise)
 
+        # if demolocation, add demo layer to map layer tree
+        if is_current_mapset_in_demolocation():
+            ltype, lname, = get_demolocation_layer_info()
+            self.GetLayerTree().AddLayer(ltype=ltype,
```

Comment:
`ltype="vector"`

Reply:
Changed.

Reply:
I think @landam means specifying the type directly on line 268, you don't need a variable for that.